resources/migrations/001_update_migrations.yaml

```diff
@@ -10540,6 +10540,37 @@ databaseChangeLog:
             sql: >-
               DELETE FROM notification WHERE internal_id IS NOT NULL;
 
+  - changeSet:
```

Comment:
the v1 views read from pulse table, now alerts is migrated to notifications so we have to upgrade them here as well.